src/bundles/peer-locations.js

```diff
@@ -25,9 +27,13 @@ export default function (opts) {
     resolvingPeers: []
   }
 
-  return {
+  return applyCache({
     name: 'peerLocations',
 
+    cacheOptions: {
+      maxAge: ms.weeks(1)
```

Comment:
So we can provide a version number here in the future if we want to invalidate existing caches, say if we totally change the structure of the data. Very cool.

https://github.com/HenrikJoreteg/redux-bundler-example/blob/0f23da51a32853a2d83aaf26e39e78f34997debc/src/utils/cache.js#L5-L14

That example also talks about adding in some unique identify where cached data is specific to a given user... we might want to cache info specific to a given ipfs node identity at some point so it's nice to know that there is a way to manage that. ✨